test/transcript/TranscriptEvent.test.ts

```diff
@@ -263,4 +264,36 @@ describe('TranscriptEvent', () => {
     expect(actualEvents[0].results[0].alternatives[0].entities[1].type).to.be.undefined;
     expect(actualEvents[0].results[0].alternatives[0].entities[1].confidence).to.eql(0);
   });
+
+  it('handles one transcript event of transcript type with language identifications', async () => {
+    const event = makeSdkTranscriptWithLanguageIdentifications();
+    const data = SdkTranscriptFrame.encode(makeSdkTranscriptFrame([event])).finish();
+    logBase64FromUint8Array(data);
+    // The steps above generates data below passed directly as test input
```

Comment:
I don't fully understand the steps above. Looks like `event` and `data` fields are not used later in the test? What are we looking for by calling `logBase64FromUint8Array` with no assertions or subsequent checks?

This looks more like two independent test cases combined into a single unit test -- encoding succeeds without error, and decoding from raw bytes succeeds. It feels awkward to combine this into a single test case.

Reply:
Calling `logBase64FromUint8Array` is only used to makes sure encoding is working correctly.

Updated assertion logic to now use `event` and compare against the decode string.

Reply:
Cool -- still feels like two test cases. Encoding, and decoding. IMO, can make it easier to reason about if each unit test to be testing a single code path.

Reply:
We cannot directly assert `logBase64FromUint8Array` with `TRANSCRIPT_EVENT_TEST_VECTORS.TRANSCRIPT_LANGUAGE_IDENTIFICATIONS` as it contains `decode(<base64 encoded string>)`.

I have split the test into individual encoding and decoding tests.